### PR TITLE
Rosanto Quality updates for VMs, Netapp, Batch Accounts, AVS

### PR DESCRIPTION
**Pull request by rodrigosantosms**
# Overview/Summary

Rosanto Quality updates for VMs, Netapp, Batch Accounts, AVS

## Related Issues/Work Items

Rosanto Quality updates for VMs, Netapp, Batch Accounts, AVS

- To associate a GitHub Issue, use a [key word](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) preceded with the GitHub issue number.
- To associate an ADO Work Item, use the key word `AB#` succeeded with the [ADO Work Item ID](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).

## This PR fixes/adds/changes/removes

1. Rosanto Quality updates for VMs, Netapp, Batch Accounts, AVS

## As part of this Pull Request I have

- [x] Read the [Contribution Guide](https://azure.github.io/Azure-Proactive-Resiliency-Library/contributing) and ensured this PR is compliant with the guide
- [x] Checked for duplicate [Pull Requests](https://github.com/Azure/Azure-Proactive-Resiliency-Library/pulls)
- [ ] Associated it with relevant [GitHub Issues](https://github.com/Azure/Azure-Proactive-Resiliency-Library/issues) or ADO Work Items (Internal Only)
- [x] Ensured my code/branch is up-to-date with the latest changes in the `main` [branch](https://github.com/Azure/Azure-Proactive-Resiliency-Library/tree/main)
- [x] Ensured PR tests are passing
- [ ] Performed testing and provided evidence (e.g. screenshot of output) for any changes associated to ARG queries and/or scripts
- [ ] Updated relevant and associated documentation (e.g. Contribution Guide, Docs etc.)

### CODE REVIEW

File: docs/content/services/compute/virtual-machines/_index.md [hunk 1]
```diff
@@ -619,3 +623,108 @@ The maintenance configuration settings allows user to schedule and manage update
...
+
+### VM-23 - Avoid using B-Series VM Sku for production VMs that need the full performance of the CPU continuously
+
+**Category: System Efficiency**
+
+**Impact: Medium**
+
+**Guidance**
+
+B-series VMs are ideal for workloads that do not need the full performance of the CPU continuously, like web servers, proof of concepts, small databases and development build environments. These workloads typically have burstable performance requirements. To determine the physical hardware on which this size is deployed, query the virtual hardware from within the virtual machine. The B-series provides you with the ability to purchase a VM size with baseline performance that can build up credits when it is using less than its baseline. When the VM has accumulated credits, the VM can burst above the baseline using up to 100% of the vCPU when your application requires higher CPU performance.
```
Comment (ejhenry) [inline docs/content/services/compute/virtual-machines/_index.md:635]:
```suggestion
B-series VMs are ideal for workloads that do not need the full performance of the CPU continuously, like web servers, proof of concepts, small databases and development build environments. These workloads typically have burstable performance requirements. To determine the physical hardware on which this size is deployed, query the virtual hardware from within the virtual machine. The B-series provides you with the ability to purchase a VM size with baseline performance that can build up credits when it is using less than its baseline. When the VM has accumulated credits, the VM can burst above the baseline using up to 100% of the vCPU when your application requires higher CPU performance. Upon consuming all the CPU credits, a B-series virtual machine is throttled back to its base CPU performance until it accumulates the credits to CPU burst again.
```

**Review by ejhenry — APPROVED** [review]
LGTM 👍

File: docs/content/services/compute/virtual-machines/_index.md [hunk 2]
```diff
@@ -28,14 +28,18 @@ The presented resiliency recommendations in this guidance include Virtual Machin
 | [VM-12 - VMs should not have a Public IP directly associated](#vm-12---vms-should-not-have-a-public-ip-directly-associated)                                                                                         | Access & Security | Medium | Verified |         Yes         |
 | [VM-13 - VM network interfaces and associated subnets both have a Network Security Group (NSG) associated](#vm-13---vm-network-interfaces-and-associated-subnets-both-have-a-network-security-group-nsg-associated) | Access & Security |  Low   | Verified |         No          |
 | [VM-14 - IP Forwarding should only be enabled for Network Virtual Appliances](#vm-14---ip-forwarding-should-only-be-enabled-for-network-virtual-appliances)                                                         | Access & Security | Medium | Verified |         Yes         |
-| [VM-15 - Customer DNS Servers should be configured in the Virtual Network level](#vm-15---dns-servers-should-be-configured-in-the-virtual-network-level)                                                            |      Storage      |  Low   | Verified |         Yes         |
```
Comment (oZakari) [inline docs/content/services/compute/virtual-machines/_index.md:31]: Can you adjust the recommendation directly as well please. It wouldnt let me add a suggestion for that line.

**Review by oZakari — CHANGES_REQUESTED** [review]
Hey Rodrigo, could you also please change the acronym from bat to ba to align to the CAF documentation: https://learn.microsoft.com/en-us/azure/cloud-adoption-framework/ready/azure-best-practices/resource-abbreviations#compute-and-web

File: docs/content/services/compute/virtual-machines/_index.md (hunk 2 above)
Comment (rodrigosantosms) [reply docs/content/services/compute/virtual-machines/_index.md:31]: I just updated the line 31.

**Comment by rodrigosantosms** [comment]
@oZakari  the BuildPr job was failing due to a change on Feb/13 where the hugo theme was changed from ACE to a new one. I had to revert it, let's discuss it offline. Please review the other changes, great that you noticed a lot of things missing. thank  you

**Comment by oZakari** [comment]
Hey Rodrigo, I had pulled it out of your branch. Looks like it may have already been synced with your local branch as it is back in. I'll pull it out.

**Review by oZakari — APPROVED** [review]

**Pull request merged**